src/conways-game-of-life/index.ts

```diff
@@ -0,0 +1,71 @@
+const getGridSize = (grid: number[][]) => {
+    const colMax = grid[0].length-1;
+    const rowMax = grid.length-1;
+    return { rowMax, colMax }
+}
+
+const updateCellStatus = (currentCellStatus: number, neighbourSum: number) => {
+    if (currentCellStatus === 1 && neighbourSum > 3) {
+        return 0;
+    }
+    else if (currentCellStatus === 1 && neighbourSum < 2) {
+        return 0;
+    }
+    else if (currentCellStatus === 0 && neighbourSum === 3) {
+        return 1;
+    }
```

Comment:
Do you need else keywords? Remember that in most of the cases it's not required. Okay okay I'm been pick again.

Reply:
Else removed!